### PR TITLE
Unified with other pages about background color and padding on both sides.

### PR DESCRIPTION
**Pull request by hiroshinishio**
## Terms of Service

|Before|
|---|
|image|

|After|
|---|
|image|

## Privacy Policy

|Before|
|---|
|image|

|After|
|---|
|image|

### CODE REVIEW

File: pages/privacy-policy.tsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ export default function PrivacyPolicy() {
           content='This is the WorkStats Privacy Policy page'
         />
       </Head>
-      <main className='px-5'>
+      <main className='bg-slate-50 px-1 md:px-16'>
```
Comment (peeyalk) [inline pages/privacy-policy.tsx:15]: tailwind has the option to make custom classes. 
you can declare them there once or change them only at one place, which will affect everything.
It's just a recommendation, not a deal breaker.

**Review by peeyalk — APPROVED** [review]

File: pages/privacy-policy.tsx (hunk 1 above)
Comment (hiroshinishio) [reply pages/privacy-policy.tsx:15]: Totally agree.
Let's try it as a refactoring in another ticket. I'll make a task in Asana.

Comment (hiroshinishio) [reply pages/privacy-policy.tsx:15]: Done.

Comment (hiroshinishio) [reply pages/privacy-policy.tsx:15]: @peeyalk

**Pull request merged**